src/GoTrueApi.ts

```diff
@@ -334,4 +415,39 @@ export default class GoTrueApi {
       return { user: null, data: null, error }
     }
   }
+
+  /**
+   * Generates links to be sent via email or other.
+   * @param type The link type ("signup" or "magiclink" or "recovery" or "invite").
+   * @param email the user's email.
```

Comment:
Can we format these with caps at the start of the description?

`@param email The user's email.`